src/Sonarr.Api.V3/ImportLists/ImportListExclusionController.cs

```diff
@@ -20,7 +20,10 @@ public class ImportListExclusionController : RestController<ImportListExclusionR
         {
             _importListExclusionService = importListExclusionService;
 
-            SharedValidator.RuleFor(c => c.TvdbId).NotEmpty().SetValidator(importListExclusionExistsValidator);
+            SharedValidator.RuleFor(c => c.TvdbId).Cascade(CascadeMode.Stop)
```

Comment:
If the `TvdbId` and title are missing won't this will lead to subsequent failures instead of reporting all issues in the initial request?

Reply:
Ahh right, only within that property, never mind me. 😄